### ISSUE TITLE
FR: show xp gained in the list of matches

### ISSUE THREAD

**Issue by z3us** [HearthSim/Hearthstone-Deck-Tracker#4259]
It would be great if we have a column to show the amount of xp gained in that match.
Thx

**Comment by njdm0001** [comment]
Yep, indeed, I do not think it's so difficult, there's also enough space to add a couple of columns more...

I think it would be very valuable to know the XP earned in each of our previous games.
